docs/medical-api/more-info/limits.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Limits and throttling"
+description: "Information about Metriport's limits related to request size and request frequency"
+---
+
+The following table outlines the current limits for certain operations on the Metriport platform.
+
+Please reach out to `support@metriport.com`.
+
+| Operation | Limit per minute |
+| Patient Create or Update | 10 |
+| Documeny Query Start  | 10 |
+| Consildated Data Query Start | 100 |
+
+The following tables outlines request size limits for certain endpoints.
+
+Coming soon...
```

Comment:
I'd leave this out and ticket writing it out

Reply:
Done.